### ISSUE TITLE
Cannot pass `max_nodes` to `Assembly`

### ISSUE THREAD

**Issue by dgruano** [BjornFJohansson/pydna#312]
Running:
`
asm = Assembly((pdonr_plasmid, insert), limit=6, max_nodes=10)
`
gives me:
`
TypeError: Assembly.__init__() got an unexpected keyword argument 'max_nodes'
`
Is this intended, or should we be able to pass `max_nodes` when instanciating a new Assembly object?

**Comment by BjornFJohansson** [comment]
Absolutely! max_nodes is passed in the assemble methods. It is a parameter that I rarely used and perhaps something that might not be needed once we have manuels new assembly code.
